hana/lib/HANAService.js

```diff
@@ -1124,6 +1124,7 @@ class HANAService extends SQLService {
 
     static OutputConverters = {
       ...super.OutputConverters,
+      LargeString: cds.env.features.sql_simple_queries === 1 ? e => `TO_NVARCHAR(${e})` : undefined,
```

Comment:
what about `cds.env.features.sql_simple_queries: 2`?
```suggestion
      LargeString: cds.env.features.sql_simple_queries > 0 ? e => `TO_NVARCHAR(${e})` : undefined,
```